### PR TITLE
qriollo (new formula)

### PR DESCRIPTION
**Pull request by pthariensflame**

### CODE REVIEW

File: qriollo.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  depends_on "ghc"
+
+  def install
+    system "make"
+    bin.install "qr"
+    (lib/"chamuyo").install "Chamuyo.q"
+  end
+
+  def caveats
+    "El módulo estándar \"Chamuyo.q\" esta en el directorio #{lib/"chamuyo"}"
```
Comment (bfontaine) [inline qriollo.rb]: Please write the caveats in English.

File: qriollo.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+class Qriollo < Formula
+  desc "Un lenguaje funcional, impuro, estricto, rioplatense y en joda."
```
Comment (bfontaine) [inline qriollo.rb]: Please write the description in English.

**Comment by pthariensflame** [comment]
@bfontaine Done!

File: qriollo.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
```
Comment (DomT4) [inline qriollo.rb]: Shouldn't need this line any more. Believe these days we enforce UTF-8 on all formula files at load.

File: qriollo.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+class Qriollo < Formula
+  desc "An impure functional language, based on Rioplatense Spanish."
+  homepage "https://qriollo.github.io"
+  url "https://qriollo.github.io/Qriollo-0.91.tar.gz"
+  sha256 "c8357af8254a082d8e4da1de1bbf13bee27cfde8adb31ea0a5a0966bfbb7b28d"
+  head "https://github.com/qriollo/qriollo.git"
+
+  depends_on "ghc"
```
Comment (DomT4) [inline qriollo.rb]: Runtime as well as buildtime?

Comment (pthariensflame) [inline qriollo.rb]: Oh, wow, that particular oversight is something *I* should have caught!  I'll fix it; thanks!

**Comment by pthariensflame** [comment]
@DomT4 Done!

**Comment by DomT4** [comment]
I've just realised you've put the file in the wrong place, heh. It needs to be placed inside `Library/Formula` inside your `$(brew --prefix)`. Seems to be in the top-level path here.

File: qriollo.rb [hunk 5]
```diff
@@ -0,0 +1,29 @@
+class Qriollo < Formula
+  desc "An impure functional language, based on Rioplatense Spanish."
```
Comment (bfontaine) [inline qriollo.rb]: Please remove the indefinite article: `desc "Impure functional ...`

File: qriollo.rb [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  depends_on "ghc" => :build
+
+  def install
+    system "make"
+    bin.install "qr"
+    (lib/"chamuyo").install "Chamuyo.q"
+  end
+
+  def caveats
+    "The standard module \"Chamuyo.q\" has been placed in #{lib/"chamuyo"}."
```
Comment (DomT4) [inline qriollo.rb]: If you wrap `def caveats` with `<<-EOS.undent` you won't need to escape the double quotes.

**Comment by pthariensflame** [comment]
@bfontaine @DomT4 All done again!  (Unless you have even more for me to do, heh.)

**Comment by DomT4** [comment]
Afraid the CI has more for you to do :trollface:. Test is failing:

```
==> /usr/local/Cellar/qriollo/0.91/bin/qr --ruta /usr/local/Cellar/qriollo/0.91/lib/chamuyo /private/tmp/qriollo20151205-65687-nfolhb/HolaMundo.q
qr: No se puede encontrar el archivo /private/tmp/qriollo20151205-65687-nfolhb/HolaMundo.q, salame.
Los directorios donde se buscan los módulos son:
    /usr/local/Cellar/qriollo/0.91/lib/chamuyo

Error: qriollo: failed
```

**Comment by pthariensflame** [comment]
@DomT4 Fixed, I hope.

**Comment by mikemcquaid** [comment]
Test still failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/37785/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_qriollo/

This may be a bit English-centric of me so other maintainers: feel free to tell me to shut up but:

given all the error messages for this software aren't in English I feel it might be better for you to maintain your own tap for this? If it breaks in future it's going to be very hard for only English speakers to debug test issues like the current one.

**Comment by pthariensflame** [comment]
Well, I'll correct this error (since I think I know how), but you may have a point.  I'll wait for the other maintwrers to chime in before doing anything else.

**Comment by bfontaine** [comment]
I agree with Mike here. I do understand the first part of the error because I learnt the basics of Spanish in high-school but almost forgotten all of it since :cry: 
There are some solutions out there to translate your project so that error messages are displayed in the user’s locale (e.g. `gettext`).

**Comment by dunn** [comment]
Short error messages are easy enough to run through Google Translate, and users submitting updates to the formula will probably know Spanish also.

**Comment by pthariensflame** [comment]
Looks like the Travis build uncovered a bug in `brew` itself.

**Comment by DomT4** [comment]
Was fixed in https://github.com/Homebrew/homebrew/commit/17dd397c5d5ba04a3ce104d05d53a2ff17edfdef. Travis is probably complaining because the build was triggered prior to that point.

**Comment by pthariensflame** [comment]
Ah, ok!

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by pthariensflame** [comment]
@bfontaine Seems like a syntax error in the `racket` formula is tripping up Travis.

**Comment by bfontaine** [comment]
@pthariensflame Thanks, that was fixed a few days ago but Travis is slow to update.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew @pthariensflame; we appreciate it! :tada:

**Comment by pthariensflame** [comment]
:)

**Pull request closed without merging**